packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -57,6 +57,25 @@
 		}
 	}
 
+	&.is-contentonly-mode {
+		background-color: $gray-900;
+		color: $white;
+		button {
+			color: $white;
+			background-color: $gray-900 !important;
+		}
+		.block-editor-block-switcher__toggle {
+			color: $white;
+		}
+		.components-toolbar-group,
+		.components-toolbar {
+			border-right-color: $gray-600 !important;
+		}
+		.is-pressed {
+			color: var(--wp-admin-theme-color);
+		}
+	}
```

Comment:
We used to have a dark toolbar which was removed here:

https://github.com/WordPress/gutenberg/pull/65204/files#diff-70b1ee574a01ebb2f11bc23c4aa937951549171851aa03e9c930310f20762de6L91

Can we reuse the styles there? It may tbe that is a better foundation to start with?

Reply:
@draganescu  added the these styles into block-tools style.scss file please review it again.